main.tf

```
@@ -159,6 +159,7 @@ module "web" {
   google_oauth_client_id       = module.keyvault.secrets["google-oauth-client-id"]
   google_oauth_client_secret   = module.keyvault.secrets["google-oauth-client-secret"]
   public_ilmo_url              = "https://${module.ilmo.fqdn}"
+  public_laskugeneraattori_url = "https://laskutus.tietokilta.fi"
```

Prefer reusing existing values where possible
```suggestion
  public_laskugeneraattori_url = "https://${module.invoicing.fqdn}"
```